src/inventory/components/Inventory.jsx

```diff
@@ -154,7 +154,7 @@ const Inventory = () => {
           ))}
         </Select>
       </SearchWrapper>
-
+      <div>{filteredList.length} {t("products_in_list")}</div>
```

Comment:
should not be seen if there are not items
<img width="337" alt="image" src="https://user-images.githubusercontent.com/29302746/183301504-badbfdc5-74a0-4125-9585-f6fe5f2ed7de.png">

Reply:
and add `margin-bottom: 17px;` to the div

Reply:
you know what? please remove it (and the translations). we need another way to show the number of items..